src/OpenTracing/NullTracer/NullSpan.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+
+namespace OpenTracing.NullTracer
+{
+    public class NullSpan : ISpan
+    {
+        internal static readonly NullSpan Instance = new NullSpan(NullSpanContext.Instance);
+
+        public ISpanContext Context { get; }
+
+        private NullSpan(ISpanContext context)
+        {
+            Context = context;
```

Comment:
why store this? why not just have `Context` above return `NullSpanContext.Instance`?